spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/HttpServerStubFactory.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.contract.stubrunner;
+
+import org.springframework.cloud.contract.stubrunner.provider.wiremock.WireMockHttpServerStub;
+
+/**
+ * Creates {@link HttpServerStub} instances.
+ */
+public class HttpServerStubFactory {
```

Comment:
Why do we need a factory? Why not just leave `new WireMockHttpServerStub()` directly?

Reply:
I was kind of hesitant to add yet another `new WireMockHttpServerStub()` call to the code base. Therefore the idea was to keep the knowledge about the default implementation in one place instead of spreading it over multiple classes, so it could potentially be replaced easier in the future. Might be overkill though :)

Reply:
Removed the factory again in PR update.